variable/technology/electricity-grid.yaml

```diff
@@ -1,25 +1,86 @@
 # List of variables related to the electricity grid
+# description of variables related to electricity demand
+ Cost of Demand-not-served|Electricity:
+   description: Cost of demand-not-served. Cost of load curtailment. Value of Lost Load (VoLL)
+   unit:  €/MWh
+
+Demand-not-served|Electricity:
+   description: Demand-not-server per region (i.e. aggregate-regions, countries, nuts1, nuts2, nuts3, or nodes).
+   unit:  GWh
 
 Network|Electricity|Demand|Reserve|Frequency Containment:
-   description: demand in frequency containment reserve (FCR)            
+   description: demand in frequency containment reserve (FCR)
    unit: MW
-   
+
 Network|Electricity|Demand|Reserve|Automatic Frequency Restoration:
-   description: demand in automatic frequency restoration reserve (AFRR)             
+   description: demand in automatic frequency restoration reserve (AFRR)
    unit: MW
-   
+
 Network|Electricity|Demand|Inertia:
-   description: demand in inertia            
+   description: demand in inertia
    unit: second
-   
+
+# description of variables related to transmission lines
+Active Power Flow|Electricity|Transmission|Line:
+   description: Active power flow through a line
+   unit: GW
+
+Capacity|Electricity|Transmission|Line:
+   description: Total transfer capacity (TTC). (maximum permissible thermal load)
+   unit: GW
+
+Impedance|Electricity|Grid:
+   description: Impedance of an interconnection between 2 regions
+   unit: S
+
+Investment|Electricity|Transmission:
+   description: Investments in transmission of the power network
+   unit: billion US$2010/yr
+
+Investment|Electricity|Transmission|Line:
+   description: Overnight investment (capital) cost in transmission lines of the power network
+   unit: billion US$2010/yr
+
+Fixed Charge Rate|Electricity|Transmission|Line:
+   description: Fixed charge rate to annualize the overnight investment cost. The Fixed Charge Rate (FCR) is the percentage of the total investment cost that is required over the project life per year to cover the minimal annual revenue requirements.
+   unit:
+
+Line Type|Electricity|Transmission|Line:
+   description: Line type: alternating current (AC) and direct current (DC)
+   unit: [AC,DC]
+
+Line Losses|Electricity|Transmission|Line:
+   description: Half ohmic losses of a line
+   unit: GW
+
+Loss Factor|Electricity|Transmission|Line:
+   description: Transmission losses equal to the line flow times this factor
+   unit:
+
```

Comment:
unit: p.u.

Reply:
A comment from @danielhuppmann was about:

> If this is just a multiplicative factor, just write unit: (i.e., no unit).

And, not fill a unit has equivalence to write "no unit".